### ISSUE TITLE
DB 检测模型：训练好的模型直接infer效果很好，export为推理模型后，漏检了很多。【急】

### ISSUE THREAD

**Issue by Smile0524** [PaddlePaddle/PaddleOCR#11782]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

各位大佬帮忙看看：

同一张图片，训练好的模型直接infer效果很好，export为推理模型后，漏检了很多。参照issues中大家的做法，把代码中DetResizeForTest注释掉，还是没有用。
export时，参数如下：

Global:
  use_gpu: false
  epoch_num: 300
  log_smooth_window: 20
  print_batch_step: 1
  save_model_dir: ../../out_model/ch_db_res18/
  save_epoch_step: 10
  # evaluation is run every 5000 iterations after the 4000th iteration
  eval_batch_step: [0, 200]
  cal_metric_during_train: false
  pretrained_model:
  checkpoints:
  save_inference_dir:
  use_visualdl: False
  infer_img: doc/imgs_en/img_10.jpg
  save_res_path: ../../out_model/ch_db_res18/predicts_db.txt

Architecture:
  model_type: det
  algorithm: DB
  Transform:
  Backbone:
    name: ResNet_vd
    layers: 18
    disable_se: True
  Neck:
    name: DBFPN
    out_channels: 256
  Head:
    name: DBHead
    k: 50

Loss:
  name: DBLoss
  balance_loss: true
  main_loss_type: DiceLoss
  alpha: 5
  beta: 10
  ohem_ratio: 3

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.001
    warmup_epoch: 2
  regularizer:
    name: 'L2'
    factor: 0

PostProcess:
  name: DBPostProcess
  thresh: 0.3
  box_thresh: 0.6
  max_candidates: 1000
  unclip_ratio: 1.5

Metric:
  name: DetMetric
  main_indicator: hmean

Train:
  dataset:
    name: SimpleDataSet
    data_dir: ../../data/seal_detect_ori/det/train
    label_file_list:
      - ../../data/seal_detect_ori/det/train.txt
    ratio_list: [1.0]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - DetLabelEncode: # Class handling label
      - IaaAugment:
          augmenter_args:
            - { 'type': Fliplr, 'args': { 'p': 0.5 } }
            - { 'type': Affine, 'args': { 'rotate': [-10, 10] } }
            - { 'type': Resize, 'args': { 'size': [0.5, 3] } }
      - EastRandomCropData:
          size: [960, 960]
          max_tries: 50
          keep_ratio: true
      - MakeBorderMap:
          shrink_ratio: 0.4
          thresh_min: 0.3
          thresh_max: 0.7
      - MakeShrinkMap:
          shrink_ratio: 0.4
          min_text_size: 8
      - NormalizeImage:
          scale: 1./255.
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: ['image', 'threshold_map', 'threshold_mask', 'shrink_map', 'shrink_mask'] # the order of the dataloader list
  loader:
    shuffle: True
    drop_last: False
    batch_size_per_card: 8
    num_workers: 4

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ../../data/seal_detect_ori/det/val
    label_file_list:
      - ../../data/seal_detect_ori/det/val.txt
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - DetLabelEncode: # Class handling label
      - DetResizeForTest:
#          limit_side_len: 960
#          limit_type: max
#           image_shape: [736, 1280]
      - NormalizeImage:
          scale: 1./255.
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: ['image', 'shape', 'polys', 'ignore_tags']
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: 1 # must be 1
    num_workers: 2

**Comment by Smile0524** [comment]
已解决，当检测和识别模型同时使用时，在所有检测模型检测到的检测框中，会依据识别结果的分数再去做一次过滤（参数：drop_score）。所以导致表象上的漏检。

**Comment by Nanh1860** [comment]
> 已解决，当检测和识别模型同时使用时，在所有检测模型检测到的检测框中，会依据识别结果的分数再去做一次过滤（参数：drop_score）。所以导致表象上的漏检。

can you tell me how to solve that? my data form is page with many data per row, i want to detect and recognize each large row but unfortunately it only can recognize only 2 row top, sometimes 3
